1. Basic Web Development/frontend/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "tsc": "tsc",
+    "tscw": "tsc --watch",
+    "wp": "webpack",
+    "wpw": "webpack --watch",
+    "serve": "http-server build"
```

Comment:
It would be nice to have a hot reload script for dev purposes 😄

Reply:
actually I don't understand why you're using http-server if you have webpack configured. You should update this serve script to use webpack dev server.